### PR TITLE
[Bulk] Validation: fixed issues

### PR DESCRIPTION
**Pull request by reemsabaw1**
## Category

- [x] Content fix
- [ ] New article

## Related issues

Global effort to fix build validation errors

## What's in this Pull Request?

Fixed missing descriptions in 7 files.

### CODE REVIEW

**Comment by VesaJuvonen** [comment]
Docs Build status updates of commit _[983b945](https://github.com/reemsabaw1/sp-dev-docs/commits/983b9450ab7603c13e4d5dd12ef755f5b1e24b3a)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/schema/document-converter.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/document-converter.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/document-converter?branch=pr-en-us-8157) | [Details](#user-content-ab531600230d7417251919cc27a00029-eb4ad6c4)
[docs/schema/document-icons-schema.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/document-icons-schema.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/document-icons-schema?branch=pr-en-us-8157) | [Details](#user-content-6370dd4d9896ec21853cfaac9279b4ff-eb4ad6c4)
[docs/schema/documentconverter-element-document-converter.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/documentconverter-element-document-converter.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/documentconverter-element-document-converter?branch=pr-en-us-8157) | [Details](#user-content-f448e4c49972168f9417749def1a1bbc-eb4ad6c4)
[docs/schema/documentlibrary-element-deploymentmanifest.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/documentlibrary-element-deploymentmanifest.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/documentlibrary-element-deploymentmanifest?branch=pr-en-us-8157) | [Details](#user-content-e754fa7fe3c9eeabf1f66cc612235e96-eb4ad6c4)
[docs/schema/documenttemplate-element-contenttype.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/documenttemplate-element-contenttype.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/documenttemplate-element-contenttype?branch=pr-en-us-8157) | [Details](#user-content-e94c7cf13b3b88807da3b4a3581141dc-eb4ad6c4)
[docs/schema/draftvisibilitytype-simple-type-deploymentmanifest.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/draftvisibilitytype-simple-type-deploymentmanifest.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/draftvisibilitytype-simple-type-deploymentmanifest?branch=pr-en-us-8157) | [Details](#user-content-ea6fa7d60dac97087bfdbdc62dcf7f61-eb4ad6c4)
[docs/schema/dropdown-element-commanduidefinitiontype-complextypeapphostwebfeatures.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/dropdown-element-commanduidefinitiontype-complextypeapphostwebfeatures.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/dropdown-element-commanduidefinitiontype-complextypeapphostwebfeatures?branch=pr-en-us-8157) | [Details](#user-content-86aeb8b20c826259900dc4c66ae10ba9-eb4ad6c4)

<a id="ab531600230d7417251919cc27a00029-eb4ad6c4"></a>
### [docs/schema/document-converter.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/document-converter.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="6370dd4d9896ec21853cfaac9279b4ff-eb4ad6c4"></a>
### [docs/schema/document-icons-schema.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/document-icons-schema.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````
 - **Line 15, Column 1**: **[Warning: yaml-duplicate-key]** `````Key 'description' is already defined, remove the duplicate key.`````
 - **Line 15, Column 14**: **[Suggestion: duplicate-descriptions - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/duplicate-descriptions?branch=main)]** `````Attribute 'description' with value 'Last modified: July 15, 2011' is duplicated in 'schema/document-icons-schema.md(15,14)', 'schema/general-schema-elements-a-h.md(10,14)', 'schema/general-schema-elements-i-z.md(10,14)', 'schema/general-schema.md(10,14)', 'schema/guid-simple-type-deploymentexportsettings.md(10,14)'....`````

<a id="f448e4c49972168f9417749def1a1bbc-eb4ad6c4"></a>
### [docs/schema/documentconverter-element-document-converter.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/documentconverter-element-document-converter.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="e754fa7fe3c9eeabf1f66cc612235e96-eb4ad6c4"></a>
### [docs/schema/documentlibrary-element-deploymentmanifest.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/documentlibrary-element-deploymentmanifest.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````
 - **Line 11, Column 1**: **[Warning: yaml-duplicate-key]** `````Key 'description' is already defined, remove the duplicate key.`````
 - **Line 11, Column 14**: **[Suggestion: duplicate-descriptions - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/duplicate-descriptions?branch=main)]** `````Attribute 'description' with value 'Last modified: March 09, 2015' is duplicated in 'schema/_username-element-accountauthcredentials-complextypesps15xsdsearchset2.md(10,14)', 'schema/accountauthcredentials-complextype-sps15xsdsearchset2.md(10,14)', 'schema/accountauthcredentials-element-sps15xsdsearchset2.md(10,14)', 'schema/active-element-source-complextypesps15xsdsearchset1.md(10,14)', 'schema/aggregations-element-deployment-manifestspview.md(10,14)'....`````

<a id="e94c7cf13b3b88807da3b4a3581141dc-eb4ad6c4"></a>
### [docs/schema/documenttemplate-element-contenttype.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/documenttemplate-element-contenttype.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````
 - **Line 76, Column 27**: **[Suggestion: disallowed-html-attribute - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/disallowed-html-attribute?branch=main)]** `````HTML attribute 'sdata' on tag 'span' isn't allowed. Replace it with approved Markdown or escape the brackets if the content is a placeholder.`````

<a id="ea6fa7d60dac97087bfdbdc62dcf7f61-eb4ad6c4"></a>
### [docs/schema/draftvisibilitytype-simple-type-deploymentmanifest.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/draftvisibilitytype-simple-type-deploymentmanifest.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````
 - **Line 11, Column 1**: **[Warning: yaml-duplicate-key]** `````Key 'description' is already defined, remove the duplicate key.`````
 - **Line 11, Column 14**: **[Suggestion: duplicate-descriptions - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/duplicate-descriptions?branch=main)]** `````Attribute 'description' with value 'Last modified: March 09, 2015' is duplicated in 'schema/_username-element-accountauthcredentials-complextypesps15xsdsearchset2.md(10,14)', 'schema/accountauthcredentials-complextype-sps15xsdsearchset2.md(10,14)', 'schema/accountauthcredentials-element-sps15xsdsearchset2.md(10,14)', 'schema/active-element-source-complextypesps15xsdsearchset1.md(10,14)', 'schema/aggregations-element-deployment-manifestspview.md(10,14)'....`````

<a id="86aeb8b20c826259900dc4c66ae10ba9-eb4ad6c4"></a>
### [docs/schema/dropdown-element-commanduidefinitiontype-complextypeapphostwebfeatures.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/dropdown-element-commanduidefinitiontype-complextypeapphostwebfeatures.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````
 - **Line 22, Column 2**: **[Suggestion: column-header-missing - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/column-header-missing?branch=main)]** `````Column headers are required for tables. Add appropriate header text, or remove any extra table cells that cause an extra column.`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/8c4da01d-e430-7fe7-0bea-2c4100cce9e8/202206131442151936-8157/BuildReport?accessString=0c8fe323caa3b01cd9acc2fe567d44eb20b9d69beafb356adbbc41bd70b7c546).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by VesaJuvonen** [comment]
Docs Build status updates of commit _[c7f5972](https://github.com/reemsabaw1/sp-dev-docs/commits/c7f5972f029191efd17cb4d2a93108c5eaabe736)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/schema/document-converter.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/document-converter.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/document-converter?branch=pr-en-us-8157) | [Details](#user-content-ab531600230d7417251919cc27a00029-72d1dcf4)
[docs/schema/document-icons-schema.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/document-icons-schema.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/document-icons-schema?branch=pr-en-us-8157) | [Details](#user-content-6370dd4d9896ec21853cfaac9279b4ff-72d1dcf4)
[docs/schema/documentconverter-element-document-converter.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/documentconverter-element-document-converter.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/documentconverter-element-document-converter?branch=pr-en-us-8157) | [Details](#user-content-f448e4c49972168f9417749def1a1bbc-72d1dcf4)
[docs/schema/documentlibrary-element-deploymentmanifest.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/documentlibrary-element-deploymentmanifest.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/documentlibrary-element-deploymentmanifest?branch=pr-en-us-8157) | [Details](#user-content-e754fa7fe3c9eeabf1f66cc612235e96-72d1dcf4)
[docs/schema/documenttemplate-element-contenttype.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/documenttemplate-element-contenttype.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/documenttemplate-element-contenttype?branch=pr-en-us-8157) | [Details](#user-content-e94c7cf13b3b88807da3b4a3581141dc-72d1dcf4)
[docs/schema/draftvisibilitytype-simple-type-deploymentmanifest.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/draftvisibilitytype-simple-type-deploymentmanifest.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/draftvisibilitytype-simple-type-deploymentmanifest?branch=pr-en-us-8157) | [Details](#user-content-ea6fa7d60dac97087bfdbdc62dcf7f61-72d1dcf4)
[docs/schema/dropdown-element-commanduidefinitiontype-complextypeapphostwebfeatures.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/dropdown-element-commanduidefinitiontype-complextypeapphostwebfeatures.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/dropdown-element-commanduidefinitiontype-complextypeapphostwebfeatures?branch=pr-en-us-8157) | [Details](#user-content-86aeb8b20c826259900dc4c66ae10ba9-72d1dcf4)

<a id="ab531600230d7417251919cc27a00029-72d1dcf4"></a>
### [docs/schema/document-converter.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/document-converter.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="6370dd4d9896ec21853cfaac9279b4ff-72d1dcf4"></a>
### [docs/schema/document-icons-schema.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/document-icons-schema.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="f448e4c49972168f9417749def1a1bbc-72d1dcf4"></a>
### [docs/schema/documentconverter-element-document-converter.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/documentconverter-element-document-converter.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="e754fa7fe3c9eeabf1f66cc612235e96-72d1dcf4"></a>
### [docs/schema/documentlibrary-element-deploymentmanifest.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/documentlibrary-element-deploymentmanifest.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="e94c7cf13b3b88807da3b4a3581141dc-72d1dcf4"></a>
### [docs/schema/documenttemplate-element-contenttype.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/documenttemplate-element-contenttype.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="ea6fa7d60dac97087bfdbdc62dcf7f61-72d1dcf4"></a>
### [docs/schema/draftvisibilitytype-simple-type-deploymentmanifest.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/draftvisibilitytype-simple-type-deploymentmanifest.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="86aeb8b20c826259900dc4c66ae10ba9-72d1dcf4"></a>
### [docs/schema/dropdown-element-commanduidefinitiontype-complextypeapphostwebfeatures.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/dropdown-element-commanduidefinitiontype-complextypeapphostwebfeatures.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/8c4da01d-e430-7fe7-0bea-2c4100cce9e8/202206131618395220-8157/BuildReport?accessString=87ea91ac16bd3a3b039be6baf99d6417c39dbcf76153629a1018174ce087c6bc).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/schema/documentconverter-element-document-converter.md [hunk 1]
```diff
@@ -104,7 +105,7 @@ The following sections describe attributes, child elements, and parent elements.
 </colgroup>
 <tbody>
 <tr class="odd">
-<td align="left"><p><span sdata="link"><a href="elements-element-document-converter.md">Elements Element (Document Converter)</a></span></p></td>
+<td align="left"><p><span><a href="elements-element-document-converter.md">Elements Element (Document Converter)</a></span></p></td>
```
Comment (RedZephyr13) [inline docs/schema/documentconverter-element-document-converter.md:108]: We need to turn this HTML table into a Markdown table.  Ping me if you need help.

File: docs/schema/documenttemplate-element-contenttype.md [hunk 2]
```diff
@@ -72,7 +73,7 @@ The following sections describe attributes, child elements, and parent elements.
 </colgroup>
 <tbody>
 <tr class="odd">
-<td align="left"><p><span sdata="link"><a href="contenttype-element-contenttype.md">ContentType Element (ContentType)</a></span></p></td>
+<td align="left"><p><span><a href="contenttype-element-contenttype.md">ContentType Element (ContentType)</a></span></p></td>
```
Comment (RedZephyr13) [inline docs/schema/documenttemplate-element-contenttype.md:76]: Same here, this needs to be turned into a Markdown table.  If we do any edits to an HTML table for disallowed HTML attributes, we need to rework the table, and turn it into a Markdown table.

File: docs/schema/documentconverter-element-document-converter.md (hunk 1 above)
Comment (reemsabaw1) [reply docs/schema/documentconverter-element-document-converter.md:108]: This one won't convert into a table for some reason.

**Comment by VesaJuvonen** [comment]
Docs Build status updates of commit _[71b4264](https://github.com/reemsabaw1/sp-dev-docs/commits/71b4264cf7481bb656c21e8a1f60ebaa47f19882)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/schema/document-converter.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/document-converter.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/document-converter?branch=pr-en-us-8157) | [Details](#user-content-ab531600230d7417251919cc27a00029-846a437d)
[docs/schema/document-icons-schema.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/document-icons-schema.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/document-icons-schema?branch=pr-en-us-8157) | [Details](#user-content-6370dd4d9896ec21853cfaac9279b4ff-846a437d)
[docs/schema/documentconverter-element-document-converter.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/documentconverter-element-document-converter.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/documentconverter-element-document-converter?branch=pr-en-us-8157) | [Details](#user-content-f448e4c49972168f9417749def1a1bbc-846a437d)
[docs/schema/documentlibrary-element-deploymentmanifest.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/documentlibrary-element-deploymentmanifest.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/documentlibrary-element-deploymentmanifest?branch=pr-en-us-8157) | [Details](#user-content-e754fa7fe3c9eeabf1f66cc612235e96-846a437d)
[docs/schema/documenttemplate-element-contenttype.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/documenttemplate-element-contenttype.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/documenttemplate-element-contenttype?branch=pr-en-us-8157) | [Details](#user-content-e94c7cf13b3b88807da3b4a3581141dc-846a437d)
[docs/schema/draftvisibilitytype-simple-type-deploymentmanifest.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/draftvisibilitytype-simple-type-deploymentmanifest.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/draftvisibilitytype-simple-type-deploymentmanifest?branch=pr-en-us-8157) | [Details](#user-content-ea6fa7d60dac97087bfdbdc62dcf7f61-846a437d)
[docs/schema/dropdown-element-commanduidefinitiontype-complextypeapphostwebfeatures.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/dropdown-element-commanduidefinitiontype-complextypeapphostwebfeatures.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/dropdown-element-commanduidefinitiontype-complextypeapphostwebfeatures?branch=pr-en-us-8157) | [Details](#user-content-86aeb8b20c826259900dc4c66ae10ba9-846a437d)

<a id="ab531600230d7417251919cc27a00029-846a437d"></a>
### [docs/schema/document-converter.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/document-converter.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="6370dd4d9896ec21853cfaac9279b4ff-846a437d"></a>
### [docs/schema/document-icons-schema.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/document-icons-schema.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="f448e4c49972168f9417749def1a1bbc-846a437d"></a>
### [docs/schema/documentconverter-element-document-converter.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/documentconverter-element-document-converter.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="e754fa7fe3c9eeabf1f66cc612235e96-846a437d"></a>
### [docs/schema/documentlibrary-element-deploymentmanifest.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/documentlibrary-element-deploymentmanifest.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="e94c7cf13b3b88807da3b4a3581141dc-846a437d"></a>
### [docs/schema/documenttemplate-element-contenttype.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/documenttemplate-element-contenttype.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````
 - **Line 67, Column 2**: **[Suggestion: column-header-missing - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/column-header-missing?branch=main)]** `````Column headers are required for tables. Add appropriate header text, or remove any extra table cells that cause an extra column.`````

<a id="ea6fa7d60dac97087bfdbdc62dcf7f61-846a437d"></a>
### [docs/schema/draftvisibilitytype-simple-type-deploymentmanifest.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/draftvisibilitytype-simple-type-deploymentmanifest.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="86aeb8b20c826259900dc4c66ae10ba9-846a437d"></a>
### [docs/schema/dropdown-element-commanduidefinitiontype-complextypeapphostwebfeatures.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/dropdown-element-commanduidefinitiontype-complextypeapphostwebfeatures.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/8c4da01d-e430-7fe7-0bea-2c4100cce9e8/202206131831150087-8157/BuildReport?accessString=a1a44733443ac077c479d04f22643a300b2c66aa997fa8626ebed3c9aa5e42b5).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Review by andrewconnell — APPROVED** [review]

**Pull request merged**